src/translations/strings.yaml

```
@@ -372,10 +372,13 @@ settings:
     value: Dark
     comment: Dark as in "dark mode theme"
 
-localNetworkAccess:
-  labelTitle: Exclude local network traffic
-  subLabel: Access Apple CarPlay, AirDrop, AirPlay and local network devices like printers.
-  disabledSubLabel: Access Apple CarPlay, AirDrop, AirPlay and local network devices like printers. The VPN must be off to edit this setting.
+bypassVPN:
+  labelTitle: Bypass VPN for select traffic
+  subLabel:
+    value: Allow RCS/MMS messaging, Apple features like CarPlay and AirDrop, and connections to local devices. May reduce network security.
+    comment: This string should be clear that major features like some messages will break, but at potential the expense of some security.
```

```suggestion
    comment: Make sure it's clear that this will allow major features like messaging apps, but potentially at the expense of some security.
```

Isn't the comment backward, given it's now bypass vs exclude?

---

As confusing as it may be, "bypass VPN" and "exclude local network traffic [from VPN]" do the same thing. We're changing how it works under the hood and changing the default, so we want to be clearer in verbage.

---

Yes that is terrible English, sorry. (I wrote that.) If we update the comment, will it update it on pontoon but not require re-translations from all our volunteers? If so (and that's the preferred path), I'll happily put up a PR to touch that up.

---

Updating the comment won't force re-translation, so it's fine to do.

---

Fix is coming: https://github.com/mozilla-mobile/mozilla-vpn-client/pull/10686